wdl2cwl/tests/wdl_files/gatk_1.wdl

```diff
@@ -34,15 +34,15 @@ task HaplotypeCaller {
         #String emitRefConfidence = if gvcf then "GVCF" else "NONE"
         Boolean dontUseSoftClippedBases = false
 
-        Array[File]+? intervalList
-        Array[File]+? excludeIntervalList
```

Comment:
Let's keep these two as they are the topic of this PR (optional array inputs) and the conversion worked.

Reply:
Ok. If I apply all the changes from https://github.com/biowdl/tasks/pull/291/files#diff-c76c01f3ca967cdb9c157a75e7fb1a08d0037543b455c2107398601a2f526ebfR1081 the resultant cwl is not accurate. Shall I keep it as it is or remove the changes?

Reply:
Can you show me what is not accurate about the resulting CWL?

Reply:
For example ```for FILE in $(inputs.sep(" ",inputBams)); do ln -s $FILE wd/\$(inputBams $FILE) ; done```